File: common/src/main/java/com/raredev/vcspace/util/ToastUtils.java
package com.raredev.vcspace.util;

import android.widget.Toast;
import com.raredev.vcspace.BaseApp;

public class ToastUtils {
  public static final int TYPE_ERROR = 0;
  public static final int TYPE_SUCCESS = 2;

  public static void showShort(CharSequence message, int type) {
    Toast.makeText(BaseApp.getInstance().getApplicationContext(), message, Toast.LENGTH_SHORT).show();
  }

  public static void showLong(CharSequence message, int type) {
    Toast.makeText(BaseApp.getInstance().getApplicationContext(), message, Toast.LENGTH_LONG).show();
  }
}

File: common/src/main/java/com/raredev/vcspace/utils/toastUtils.kt
package com.raredev.vcspace.utils

import android.content.Context
import android.widget.Toast

fun showShortToast(context: Context, text: CharSequence) {
  Toast.makeText(context, text, Toast.LENGTH_SHORT).show()
}

fun showLongToast(context: Context, text: CharSequence) {
  Toast.makeText(context, text, Toast.LENGTH_LONG).show()
}
